packages/bsb/docs/components/components-toc.rst

```diff
@@ -2,12 +2,12 @@
 
 .. _main-components:
 
-Domains
-=======
+Components
+==========
```

Comment:
This is the only place (the sidebar, right?) where I do actually want it to be called "Domains"

Reply:
I like it! But could you reflow it so that the tutorial on how to write components comes at the start, maybe something like:

> For a step by step guide to write your own component, take a look at the [tutorial](link). Here is how you do it (theoretically):

Reply:
I prefer to have first a section to tell the user about the existing components and a section about reaching for help before the coding part. But I brought up this link to the tutorial